platform-tests/src/test/java/org/junit/platform/console/options/StdStreamTest.java

```diff
@@ -0,0 +1,11 @@
+package org.junit.platform.console.options;
+import org.junit.jupiter.api.Test;
+
+public class StdStreamTest {
+
+	@Test
+	void printTest() {
```

Comment:
I don't really like this solution. Any ideas on how to make this test cleaner is well appreciated.

Reply:
I think it's fine to have a test case that only exists for the sake of another tests. However, please name it `StdStreamTestCase` (as per our convention) so it doesn't get picked up as a regular test case.